backdoor/pom.xml

```diff
@@ -66,12 +64,7 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Vendor>Codice</Bundle-Vendor>
                         <Bundle-Activator>org.codice.acdebugger.backdoor.Backdoor</Bundle-Activator>
-                        <Embed-Dependency>acdebugger-common, boon</Embed-Dependency>
-                        <Import-Package> <!-- imports specific as optional for embedding boon -->
```

Comment:
This makes me happy.

Reply:
Me too, I was not a fan of being forced to do that. But, to Boon's defense, it wasn't its fault but more the fault of the bundle plugin.